### PR TITLE
style(frontend): update Hero buttons to new compact style

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

We update the new `ButtonHero` style and we use it throughout the buttons in the `Actions` component.

Screenshot 2024-07-23 at 11 47 36
Screenshot 2024-07-23 at 11 47 47
Screenshot 2024-07-23 at 11 47 53
Screenshot 2024-07-23 at 11 48 19
Screenshot 2024-07-23 at 11 48 51
Screenshot 2024-07-23 at 11 49 04
Screenshot 2024-07-23 at 11 49 24

### CODE REVIEW

File: src/frontend/src/lib/components/receive/ReceiveButton.svelte [hunk 1]
```diff
@@ -2,14 +2,10 @@
...
 	import IconReceive from '$lib/components/icons/IconReceive.svelte';
 	import { i18n } from '$lib/stores/i18n.store';
-	import { IconQRCodeScanner } from '@dfinity/gix-components';
-	import ButtonDualMode from '$lib/components/ui/ButtonDualMode.svelte';
-
-	export let compact = false;
+	import ButtonHero from '$lib/components/ui/ButtonHero.svelte';
 </script>
 
-<ButtonDualMode on:click disabled={$isBusy} alt={$i18n.receive.text.receive} {compact}>
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/receive/ReceiveButton.svelte:11]: The component `ButtonDualMode` can be deleted?

**Review by peterpeterparker — APPROVED** [review]
LGTM, thanks

Unsollicited feedback: from having a look at the screenshots, something in the aligment feels off. But I'm guessing there will be other PRs after this one.

File: src/frontend/src/lib/components/receive/ReceiveButton.svelte (hunk 1 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/receive/ReceiveButton.svelte:11]: Indeed! But I will do it in another PR

**Comment by AntonioVentilii-DFINITY** [comment]
> Unsollicited feedback: from having a look at the screenshots, something in the aligment feels off. But I'm guessing there will be other PRs after this one.


Correct, the token logo is not central, but that will be in another PR

File: src/frontend/src/lib/components/receive/ReceiveButton.svelte (hunk 1 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/receive/ReceiveButton.svelte:11]: Done with PR #1833

**Pull request merged**